galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,210 @@
+from galaxy_ng.app.access_control import access_policy
+from random import sample
+from rest_framework.response import Response
+from pulp_ansible.app.models import CollectionVersion, AnsibleDistribution
+from galaxy_ng.app.models import Namespace
+from galaxy_ng.app import settings
+from galaxy_ng.app.api import base as api_base
+
+
+class LandingPageView(api_base.APIView):
+    permission_classes = [access_policy.LandingPageAccessPolicy]
+
+    def get(self, request, *args, **kwargs):
+        golden_name = settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
+
+        distro = AnsibleDistribution.objects.get(base_path=golden_name)
+        repository_version = distro.repository.latest_version()
+        versions = CollectionVersion.objects.filter(pk__in=repository_version.content)
```

Comment:
this is gonna return the number of collection versions, not the number of collections. `versions = CollectionVersion.objects.filter(pk__in=repository_version.content, is_highest=True)` should do the trick